src/ScheduleManager.cpp

```diff
@@ -392,6 +394,7 @@ bool ScheduleManager::RunTaskletsForTime( long long timeout )
 
 bool ScheduleManager::RunNTasklets( int n )
 {
+	TelemetryZone telemetryZone(TMCM_CPP, "ScheduleManager::Run()", __FILE__, __LINE__, tracy::Color::LightGreen);
```

Comment:
As this function calls ScheduleManager::Run which also defines a Telemetry zone does this lead to double zone marking?

I see that this pattern is also missing from ScheduleManager::RunTaskletsForTime which is the game entry.

Reply:
Good points. I am ambivalent about instrumenting the wrappers around `ScheduleManager::Run`. There is not that much extra stuff going on, so any runtime difference is likely going to be marginal. Therefore, the only immediate benefit of instrumenting the wrappers as well is to know which of them may have triggered the `ScheduleManager::Run` call.

Reply:
If keeping the wrapper markup, and without really studying the docs for Telemetry, should `ScheduleManager::Run()` be `ScheduleManager::RunNTasklets` ?